test/jdk/java/awt/image/VolatileImage/ReportRenderingError.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.GraphicsEnvironment;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.awt.image.ImageObserver;
+import java.awt.image.ImageProducer;
+
+/*
+ * @test id=default
+ * @bug 8272288
+ * @key headful
+ * @summary Broken rendering should be reported by the contentsLost()
+ *
+ * @run main/othervm ReportRenderingError
+ */
+
+/*
+ * @test id=windows
+ * @bug 8272288
+ * @key headful
+ * @summary Broken rendering should be reported by the contentsLost()
+ * @requires (os.family == "windows")
+ *
+ * @run main/othervm -Dsun.java2d.opengl=True ReportRenderingError
+ * @run main/othervm -Dsun.java2d.d3d=True    ReportRenderingError
+ * @run main/othervm -Dsun.java2d.d3d=false   ReportRenderingError
+ */
+
+/*
+ * @test id=macos
+ * @bug 8272288
+ * @key headful
+ * @summary Broken rendering should be reported by the contentsLost()
+ *
+ * @requires (os.family == "mac")
+ * @run main/othervm -Dsun.java2d.opengl=True ReportRenderingError
```

Comment:
I ran this test in macOS and without the fix it exits after checking only OpenGL pipeline. We dont override restoreContents() in CGLVolatileSurfaceManager/MTLVolatileSurfaceManager so its better if we can force the test to run on all pipelines on individual platforms and not exit on first failure based on jtreg instrcutions. This will make the test more robust.